src/db/contractEventLogs/getContractEventLogs.ts

```diff
@@ -148,22 +150,22 @@ export const getEventLogsByCursor = async ({
                 { createdAt: { gt: cursorObj.createdAt } },
                 {
                   createdAt: { equals: cursorObj.createdAt },
-                  chainId: { gt: cursorObj.chainId },
```

Comment:
this feels accidental, we shouldn't need to compare `gt` for chainId right?

Reply:
Actually, leave this as "gt" (even though the behavior is different with strings). It's a safer change than updating this to equals. If using eq here it matches many more rows than it would before.